.github/workflows/android.yml

```diff
@@ -93,10 +96,14 @@ jobs:
           This build is based on http://github.com/subsurface/subsurface/commit/${{ github.sha }}
 
           None of these artifacts are signed.
-          The Android APK can be side-loaded on most Android devices. If you had a previous Subsurface-mobile version installed from the Google Play store, you'll have to uninstall that first.
+          The Android APK can be side-loaded on most Android devices.
```

Comment:
So that assumes that I add the existing keystore / password as secrets, correct?

Reply:
I sent you an email about that. I'd probably generate a new key and keystore dedicated to CICD releases - there doesn't seem to be a downside to using self signed keys for this.